### ISSUE TITLE
V3.3.0 typeToBooleanConverter

### ISSUE THREAD

**Issue by CarlosAdrianM** [xceedsoftware/wpftoolkit#1298]
This code works fine in V3.2.0:
```

            <xcdg:DataGridControl.View>
                <xcdg:TableView UseDefaultHeadersFooters="False">
                    <xcdg:TableView.FixedHeaders>
                        <DataTemplate>
                            <xcdg:ColumnManagerRow/>
                        </DataTemplate>
                    </xcdg:TableView.FixedHeaders>
                </xcdg:TableView>
            </xcdg:DataGridControl.View>
```

But in V3.3.0 it throws an exception because Can't find the resource 'typeToBooleanConverter'.

Sorry if it is a duplicate one, but I was not able to find a similar issue.

**Comment by XceedBoucherS** [comment]
Hi, This is already fixed in v3.4. The problem is related to the TableView.
You can see the "What's New" from every releases here : https://xceed.com/xceed-toolkit-plus-for-wpf/

This is related to https://github.com/xceedsoftware/wpftoolkit/issues/1282

**Comment by jeffdavis503** [comment]
If this is fixed in 3.4 then how do I get version 3.4?  I only see 3.3 available?

**Comment by s2quake** [comment]
Could you tell me when v2.4 will be released?

**Comment by simeyla** [comment]
@jeffdavis503 The free version is behind and as far as I can see is released every 3 months - making the next version approx. on June 5th.

See https://github.com/xceedsoftware/wpftoolkit
https://github.com/xceedsoftware/wpftoolkit/wiki/Xceed-Toolkit-Plus-for-WPF 

That is why you don't see it. You need the plus edition to get - well that version is currently 3.6.

So I haven't updated the toolkit in 2 years - hit up against this regression bug today - and then find I can't even downgrade it to 3.2 within nuget.

Uninstalling and reinstalling with this command worked (to 3.2), but you can't use the GUI since it's only the latest version there.

     Install-Package Extended.Wpf.Toolkit -Version 3.3.0

**Comment by XceedBoucherS** [comment]
Hi,

Currently, 
-users of the Plus version have v3.6 (v3.7 is about to be released in the next weeks)
-users of the Free version have v3.3.
Users of the Plus version always have the latest version, more features and fast support. You can try the Plus version for 45 days for free here : https://xceed.com/xceed-toolkit-plus-for-wpf/

The free version goes up at about the same time as the Plus version goes up, but not always. 
By the way, Nuget now offers the old versions of the Free Toolkit.

**Comment by jeffdavis503** [comment]
The problem is there is a bug in the v3.3 Free version that was fixed in
the v3.4 version which is in the Plus version.  I can't use the v.3.3 free
version because of the DataGrid bug so I rolled back to the v.3.2 Free
version.  The problem is there were many improvements in the DataGrid in
3.3 that I would like to take advantage of but can't because of this bug.

**Comment by XceedBoucherS** [comment]
Hi,

As said earlier, issue #1282 proposes a fix for this until v3.4 Community edition is released.
Thank you.
